src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -1311,7 +1311,9 @@ private void Initialized(object message)
             if (message is GossipEnvelope)
             {
                 var ge = message as GossipEnvelope;
-                ReceiveGossip(ge);
+                var receivedType = ReceiveGossip(ge);
+                if(_cluster.Settings.VerboseGossipReceivedLogging)
+                    _log.Debug("Cluster Node [{0}] - Received gossip from [{1}] which was {2}.", _cluster.SelfAddress, ge.From, receivedType);
```

Comment:
Check the setting and log how we processed the gossip if enabled, on `DEBUG` loglevel